lib/spack/spack/reporters/cdash.py

```diff
@@ -166,6 +177,11 @@ def clean_log_event(event):
                             event['source_file'])
                     return event
 
+                # Convert errors to warnings if the package reported success.
+                if package['result'] == 'success':
```

Comment:
Nice that there was a simple way to detect whether or not the installation was successful.